tools/misc/tpm2_checkquote.c

```diff
@@ -5,7 +5,9 @@
 #include <stdlib.h>
 #include <string.h>
 
- #include <openssl/pem.h>
+#include <inttypes.h>
```

Comment:
Group this with other C standard headers and add this to the top to list them alphabetically.